src/services/sync/strategies/LDAPSyncer.js

```diff
@@ -55,17 +55,27 @@ class LDAPSyncer extends Syncer {
 	}
 
 	async processLdapUsers(school) {
-		syncLogger.info(`Getting users for school ${school.name}`, { syncId: this.syncId });
-		const ldapUsers = await this.ldapService.getUsers(this.system.ldapConfig, school);
-		this.stats.users += ldapUsers.length;
-		await this.sendLdapUsers(ldapUsers, school.ldapSchoolIdentifier);
+		try {
+			syncLogger.info(`Getting users for school ${school.name}`, { syncId: this.syncId });
+			const ldapUsers = await this.ldapService.getUsers(this.system.ldapConfig, school);
+			this.stats.users += ldapUsers.length;
+			await this.sendLdapUsers(ldapUsers, school.ldapSchoolIdentifier);
+		} catch (err) {
+			this.logError('Error while syncing', { error: err, syncId: this.syncId });
+			this.stats.errors.push(err);
+		}
 	}
 
 	async processLdapClasses(school) {
-		syncLogger.info(`Getting classes for school ${school.name}`, { syncId: this.syncId });
-		const ldapClasses = await this.ldapService.getClasses(this.system.ldapConfig, school);
-		this.stats.classes += ldapClasses.length;
-		await this.sendLdapClasses(ldapClasses, school);
+		try {
+			syncLogger.info(`Getting classes for school ${school.name}`, { syncId: this.syncId });
+			const ldapClasses = await this.ldapService.getClasses(this.system.ldapConfig, school);
+			this.stats.classes += ldapClasses.length;
+			await this.sendLdapClasses(ldapClasses, school);
+		} catch (err) {
+			this.logError('Error while syncing', { error: err, syncId: this.syncId });
+			this.stats.errors.push(err);
```

Comment:
```suggestion
			syncLogger.error('processLdapClasses', { error: err, syncId: this.syncId });
			this.stats.errors++;
```

Reply:
What do you think about?
And thank you about the hint with syncLogger.error.

```suggestion
			syncLogger.error('Error while syncing process Ldap Classes', { error: err, syncId: this.syncId });
			this.stats.errors.push(err);
```